package.json

```diff
@@ -58,7 +75,8 @@
     "watch": "npm run watch:js & npm run watch:scss",
     "watch:js": "webpack -d --watch",
     "watch:scss": "nodemon -e src/scss -x \"npm run build:scss\"",
-    "test:js": "mocha --timeout 30000 --recursive test/",
+    "test": "node_modules/.bin/karma start karma.config.js",
```

Comment:
`npm` automatically puts `node_modules/.bin` in the path for `npm run` scripts. So you should be able to define this as `karma start karma.config.js` directly.